### DISCUSSION TITLE
What is the correct use of parentOrganization and subOrganization?

### DISCUSSION THREAD

**Discussion by alvaropicho** [schemaorg/schemaorg#3316] [Q&A] [answered]
Until I'm not an experienced in the creation of Schemas, a doubt arises to me, and I believe that I can raise well.

When I make use of the parentOrganization property, one of these 2 readings/interpretations is valid, but I'm not sure which one is correct.

First, the schema, when I write:
{
		"@type": "Organization",
		"@id": "https://website.es/#organization",
		"name": "Organization 1",
		"legalName": "legal name company",
		"url": "https://site.com/",
		"logo":".....png",
		"parentOrganization":
			{
				"@type": "OnlineBusiness",
				"name": "name1",
				"legalName": "legal name company",
....

My dount, Am I declaring...:

1) That Organization 1 **IS** the parentOrganization of an OnlineBusiness.

So I subordinate OnlineBusiness to Organization. Is this correct?

2) Or is the statement the other way around: that Organization 1 **HAS** a parent Organization that is an Online Business.

Semantically the question is, which is correct.

1) This **IS** that

Or

2) This **HAS** that

I think the correct is 1). Am I wrong?

Now the case with subOrganization.

My Case is a Research Center of a College, one of many over the College.

First I built:

Research Center -> ParentOrganization -> College

Because I think the sentence was: 

Research Center **HAS** a parent Organization that is the College.

But days after, I think I'm wrong and i think the best Schema is:

Reseach Center -> sub Organization -> College

So the sentence is:

The Research Center **IS** a sub Organization of the College.

And I close the circle, which is the correct lecture with both types (parent and sub)?

1) This IS that
2) This HAS that

I think, nº 1 is the correct but I'm not sure... 

Any advice will be very welcome.

With love & respect,

Álvaro

**Comment by RichardWallis** [comment, marked as answer]
The sentence construct I would use is:

> Organization A's _parentOrganization_ is Organization B

In the same way that

> Organization A's _founder_ is A. Person

or

> Organization A's _telephone_ is 08000 111 11111

Coming at it from the other direction...

> Organization B's _subOrganization_ is Organization A

**Reply by alvaropicho** [reply]
Thank you very much @RichardWallis 

Your answer will help me over all schema's types and properties.

Even, this sentence construct has a stronger logical, easy to manage, because a property/value, is held, it has.

For me, from now on I'll tell myself: THIS **HAS** _this property_ that **IS** THAT _type_

I'm really happy, thank you again Richard
